test/libsolidity/semanticTests/externalContracts/ramanujan_pi.sol

```diff
@@ -35,10 +35,10 @@ contract test {
 // compileViaYul: also
 // ----
 // constructor()
-// gas irOptimized: 500656
+// gas irOptimized: 527609
```

Comment:
Looks like the extra code doesn't always get optimized out completely. This looks like a pretty significant increase in gas usage.